apps/ui-patterns/src/app/UIPatternsSection/UIPatternsSection.module.css

```diff
@@ -1,6 +1,6 @@
 @import url('@tacc/core-styles/dist/settings/border.css');
 @import url('@tacc/core-styles/dist/settings/space.css');
-@import url('@tacc/core-styles/dist/settings/font.css');
+@import url('@tacc/core-styles/dist/settings/font--portal.css');
```

Comment:
This is not doing anything yet, and may never, but it follows the same logic as the color fix in many other files. Read about that.